### ISSUE TITLE
pp_check fails on objects with attributes

### ISSUE THREAD

**Issue by iamamutt** [stan-dev/rstanarm#168]
#### Summary:

If a numeric vector has an attribute, pp_check fails.

#### Description:

I'm working with the haven package and it adds attributes to data vectors. Some functions will fail if the data is not coerced to numeric or attributes removed.

#### Reproducible Steps:

```
data <- mtcars
attr(data$mpg, 'junk') <- 0

fit <- stan_glmer(mpg ~ wt + am + (1|cyl), data = data,
                  iter = 400, chains = 2)

pp_check(fit)

> Error in validate_y(y) : 'y' must be a vector or 1D array.
```

#### RStanARM Version:

2.14.1

#### R Version:

3.3.2

#### Operating System:

Windows 10 64-bit

**Comment by jgabry** [comment]
Thanks for reporting this! Should be easy for us to fix.

**Comment by jgabry** [comment]
Should be fixed by 12da51aa8672c538f96bb1038e79780a1fa82445
